### PR TITLE
Add demo deploy workflow for components demo

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:** 

**Description of changes:**
Add a deploy script to deploy our demo app every time we push to master branch. 

**Testing**
1. Have you successfully run `npm run build:release` locally? yes

2. How did you test these changes? Verified that the github action deploys the demo correctly.

3. If you made changes to the component library, have you provided corresponding documentation changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by devalevenkatesh — DISMISSED** [review]

**Review by ltrung — COMMENTED** [review]
I do not think this is what we are looking for. The task is to deploy a demo for QA testing as part of the release script (similar to JS SDK).

**Review by ltrung — CHANGES_REQUESTED** [review]
Need to update the release script to match JS SDK (and remove verdaccio)

**Comment by devalevenkatesh** [comment]
> I do not think this is what we are looking for. The task is to deploy a demo for QA testing as part of the release script (similar to JS SDK).

So the JS SDK also deploys the demo as part of the github action trigger on a PR which this PR addresses, I thought this commit is getting aligned to that. Looks like that is not what is intended. Correct @ltrung ?

File: demo/meeting/serverless/deploy.js [hunk 1]
```diff
@@ -114,14 +114,16 @@ function appHtml(appName) {
 
 function ensureApp(appName = app) {
   console.log(`Building ${appName} application`);
-  spawnOrFail('npm', ['run', 'build'], { cwd: path.join(__dirname, '..') });
-  spawnOrFail('npm', ['run', 'build:prod'], { cwd: path.join(__dirname, '..') });
+  const output = spawnOrFail('npm', ['run', 'build'], { cwd: path.join(__dirname, '..') });
+  console.log(output);
```
Comment (ltrung) [inline demo/meeting/serverless/deploy.js:118]: Can we pass in a flag to suppress this instead?

Comment (michhyun1) [reply demo/meeting/serverless/deploy.js]: Done

**Comment by michhyun1** [comment]
Verdaccio is already removed in a previous PR. This PR should deploy the demo app at the end of the release script

**Review by ltrung — APPROVED** [review]

**Pull request merged**